specviz/widgets/workspace.py

```diff
@@ -1,6 +1,7 @@
-import logging
 import os
 import sys
+import logging
+from collections import OrderedDict
```

Comment:
Dictionaries are ordered by default now.

Reply:
I'm being overly cautious here since we continue to support Py36, where that is only an implementation detail of CPython. It only became a language standard in Py37.